libbz2-rs-sys/src/decompress.rs

```diff
@@ -1116,15 +1117,22 @@ pub(crate) fn decompress(
                 }
             }
             if current_block == Block46 {
-                if N >= 2 * 1024 * 1024 {
+                if (1 << logN) >= 2 * 1024 * 1024 {
```

Comment:
Maybe `1i32 << logN` to clarify that the output is a u32? Or alternatively directly check the value of `logN`.

Reply:
sure, we're now comparing the log values